DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/README.md

```diff
@@ -0,0 +1,317 @@
+      > 
+      > ![](assets/fix-reset_n-platform-designer.png)
+
+   4. Save the system by clicking `File` > `Save`
+
+   5. Generate the syste so we can include it in the Intel® Quartus® Prime project by clicking `Generate HDL...`
+
+      ![](assets/generate-hdl.png)
+   
+   6. Close Platform Designer. 
+   
+6. In the Intel® Quartus® Prime window, run Analysis and Elaboration by clicking 'Start Analysis and Elaboration'.
+
+   ![](assets/start-analysis.png)
+
+7. Now, we will select pins for the `i_clk` and `reset_button_n` inputs and `fpga_led` output. The JTAG IP will handle the connection between your design and the JTAG pins on your board automatically.
+
+   1. Open the pin planner using `Assignments` > `Pin Planner` in the main Intel® Quartus® Prime GUI. Consult the data sheet for your board to choose an appropriate clock input. In this project, the `PIN_AM10` was chosen because it is used for supplying a 100MHz clock signal in the the GHRD source code (see link in **Board-specifc Considerations**).
+
+   2. Assign pins for the `fpga_led` and `reset_button_n` signals using the same methodology:
+   
+      *Pin planner from GHRD:*
+
+      ![](assets/pins-from-ghrd.png)
+
+      *Final pin planner configuration:*
+
+      ![](assets/pins-from-design.png)
+
+8. Now add the timing constraints. 
+
+   1. If you are using the Intel® Arria® 10 SX SoC Dev Kit, you can find a timing constraints file for the JTAG interface (jtag.sdc) in the GHRD. This file was added during project creation.
+
+   2. Create a new Synopsis Design Constraints (SDC) file named `add.sdc` and insert a new clock called `i_clk` to match the clock you defined in `add.sv`. Set the period to be 10ns:
+
+      ```
+      set_time_format -unit ns -decimal_places 3
+      create_clock -name i_clk -period 10 [get_ports {i_clk}]
+      ```
+
+   3. Cut the clock paths for asynchronous I/O:
+      
+      ```
+      set_false_path -from [get_ports {reset_button_n}] -to * 
+      set_false_path -from [get_ports {fpga_led}] -to *
+      set_false_path -from * -to [get_ports {fpga_led}]
+      ```
+
+9. Compile the full design by clicking the 'Start Compilation' button in the Intel® Quartus® Prime GUI.
+
+      ![](assets/start-compilation-quartus.png)
+
+10. Copy the generated `add.sof` file to the `system_console` directory.
+
+### Additional Documentation
+- [Intel® Arria® 10 SoC Golden System Reference Design](https://rocketboards.org/foswiki/Documentation/Arria10SoCGSRD) describes a reference design you can use with your Intel® Arria® 10 SX SoC Developer kit.
+- [Intel® Arria 10 SX SoC Development Kit](https://www.intel.com/content/www/us/en/products/details/fpga/development-kits/arria/10-sx.html) describes the Intel® Arria® 10 SX SoC Development kit in greater detail.
+- [Intel® FPGA Software Installation and Licensing](https://www.intel.com/content/www/us/en/docs/programmable/683472/current/faq.html) describes how to license Intel® Quartus® Prime Pro software.
```

Comment:
You can end the Quartus docs URLs after "current/". That way, if the top-level page ever changes, you still get taken to the first page of the current doc.